tests/80-algorithms-jcs.js

```diff
@@ -24,7 +25,7 @@ const {match} = endpoints.filterByTag({
 });
 const should = chai.should();
 
-describe('Algorithms - Transformation (eddsa-jcs-2022)', function() {
+describe('eddsa-jcs-2022 - Algorithms - Transformation', function() {
```

Comment:
Good move. I like the cryptosuite name first. Thanks!

Reply:
it matches the original better, perhaps we should make a style guide and ensure cryptosuite is first? Data Integrity still lists itself first. Also, the sections are nice, but are not necessarily sorted alphabetically in the report which is a little annoying.